docs/mining/proof-of-work/pool-mining/gominer/linux.md

```diff
@@ -0,0 +1,99 @@
+$ wget https://github.com/decred/decred-binaries/releases/download/v1.0.0_gominer/gominer-linux-amd64-opencl-v1.0.0.tar.gz
+$ wget https://github.com/decred/decred-binaries/releases/download/v1.0.0_gominer/gominer-linux-amd64-opencladl-v1.0.0.tar.gz
+```
+
+>Extract the .tar.gz File
+
+Extract or copy all of the files to a new directory. For this tutorial we will use `~/decred/` as our working directory. From the command line:
+
+```no-highlight
+$ tar -xf gominer-linux-amd64-cuda-v1.0.0.tar.gz -C ~/decred/
+```
+
+Substitute the filename for the one you downloaded and the directory for where you want it installed.  When you unpack the tar, it will create a directory named `gominer-linux-amd64-opencladl-v1.0.0`. The contents of this directory should be `blake256.cl`, `blake256-old.cl`, `gominer`, `LICENSE`, `README.md`, and `sample-gominer.conf`.
+
+For the CUDA version, the CUDA runtime library is now included, so you no longer need to download the 1GB SDK from Nvidia.  The files in this version will also include `libcudart.so.8.0` and `rungominer.sh`. The script is used to start gominer with the included runtime library for your convenience.  You must have drivers installed for your graphics card that support CUDA 8, however.  Older drivers will cause gominer to crash on launch.
+
+### **Set Up the *gominer* Configuration File**
+
+> Step 1: Choose Mining Pool
+
+Before we begin this step, it is important that you've already signed up for a mining pool account (if required by your chosen mining pool, you'll have to create a worker at the pool's website). Record your worker's login and password, and the pool's stratum+tcp address:port for later use.
+
+If this is new to you, here are a few guides for creating workers on some of the Decred mining pools (WARNING: do not forget your PIN you set during account creation - you will be unable to withdraw any mined DCR without it):
+
+- [https://dcr.maxminers.net/index.php?page=gettingstarted](https://dcr.maxminers.net/index.php?page=gettingstarted)
+- [https://dcr.suprnova.cc/index.php?page=gettingstarted](https://dcr.suprnova.cc/index.php?page=gettingstarted)
+- [https://www2.coinmine.pl/dcr/index.php?page=gettingstarted](https://www2.coinmine.pl/dcr/index.php?page=gettingstarted)
+- [https://pool.mn/dcr/index.php?page=gettingstarted](https://pool.mn/dcr/index.php?page=gettingstarted)
+
+> Step 2: Verify Device ID#
+
+Before setting up the configuration file, it is vital you find out which devices, as identified by gominer, you wish to use to mine (especially if you want to avoid mining with your CPU or IGP). To do this, open a terminal window, navigate to the location you installed gominer above (e.g. `~/decred/gominer-linux-amd64-cuda-v1.0.0`), type `./gominer -l` or `./rungominer.sh -l` if you're using the CUDA version, press Enter, and a list of the devices you can mine with will be displayed. Record which device IDs you would like to use. Hint: Your video cards may be listed by their codename.
```

Comment:
Does gominer even let you do cpuu mining without extra effort?  I thought not (but I could be missremembering).  If it requiers extra options to use cpu mining (so one would not hit it by accident) I wouldn't mention it here.